robottelo/cli/factory.py

```diff
@@ -1691,9 +1691,15 @@ def setup_org_for_a_custom_repo(options=None):
     cvv = ContentView.info({'id': cv_id})['versions'][-1]
     # Promote version to next env
     try:
-        ContentView.version_promote(
-            {'id': cvv['id'], 'organization-id': org_id, 'to-lifecycle-environment-id': env_id}
-        )
+        all_lifecycle_env = ContentView.info({'id': cv_id})['lifecycle-environments']
```

Comment:
We're running same ContentView.info command for getting cvv and LCE, if you could store in some variable first and use it for CVV and LCE, then we don't run it again